client/state/signup/verticals/reducer.js

```diff
@@ -1,13 +1,25 @@
 /** @format */
+/**
+ * External dependencies
+ */
+import { toLower, trim } from 'lodash';
 
 /**
  * Internal dependencies
  */
-import { createReducer, keyedReducer } from 'state/utils';
+import { createReducer } from 'state/utils';
 import { SIGNUP_VERTICALS_SET } from 'state/action-types';
 
-const verticals = createReducer( null, {
-	[ SIGNUP_VERTICALS_SET ]: ( state, action ) => action.verticals,
-} );
+const verticals = createReducer(
+	{},
+	{
+		[ SIGNUP_VERTICALS_SET ]: ( state, action ) => {
+			return {
+				...state,
+				[ trim( toLower( action.search ) ) ]: action.verticals,
```

Comment:
It's just curious. Why do you use lodash functions instead of built in ones such as `.toLowerCase()` and `.trim()`?

Reply:
Taggon beat me to it :)
I was going to suggest that we use the built-in function whenever possible.

Reply:
> Why do you use lodash functions

I did at first, but lodash does all the null checks for us. I can revert them and check for string values. Probably worth it. 

Cheers!